### PR TITLE
Done

### PR DESCRIPTION
**Pull request by aarrtemm**
[DEMO LINK](https://aarrtemm.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://aarrtemm.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  padding: 0;
```
Comment (SanyaBratashchuk) [inline src/style.css:16]: do not apply any rules to the `*` selector except `box-sizing`

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
```
Comment (SanyaBratashchuk) [inline src/style.css:21]: move it to the `html` tag selector and let inputs inherit it

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+.search-form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+label {
+  cursor: default;
+}
```
Comment (SanyaBratashchuk) [inline src/style.css:46]: use classes

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 3 above)
Comment (aarrtemm) [reply src/style.css]: I changed everything

File: src/index.html [hunk 4]
```diff
@@ -2,15 +2,42 @@
...
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="search-form search-form-big" 
+      data-qa="big"
+      method="post"
+      action="/"
```
Comment (alexandra-protyanova) [inline src/index.html:16]: u have correct action in repo to this task

**Review by alexandra-protyanova — APPROVED** [review]

**Pull request closed without merging**